docs/tutorials/nav2-integration.md

```diff
@@ -1,103 +1,83 @@
 # Using Beluga with Nav2
 
-:::{figure} ../_images/beluga_nav2.gif
-:alt: Short video of the integration of beluga_amcl with nav2.
-:::
-
 ## Overview
 
 [Nav2](https://docs.nav2.org/index.html) is a ROS 2 framework that includes several components for robot navigation. One of these components is the localization system, typically [`nav2_amcl`](https://github.com/ros-navigation/navigation2/tree/humble/nav2_amcl).
 
-This tutorial will demonstrate how to use [`beluga_amcl`](../packages/beluga_amcl/docs/index.md) in place for `nav2_amcl`.
-
-To keep it simple, we have built this tutorial on top of the [Nav2 Getting Started Guide](https://docs.nav2.org/getting_started/index.html#). We will use the same simulated [TurtleBot 3](https://emanual.robotis.com/docs/en/platform/turtlebot3/quick-start/) robot and environment.
+This tutorial will demonstrate how to use [`beluga_amcl`](../packages/beluga_amcl/docs/index.md) instead of `nav2_amcl` for a simulated TurtleBot 3 on ROS 2 Humble. You may also try Beluga on other supported ROS 2 versions, with different types of robots.
 
-## Requirements
+## Bring the code
 
-:::{important}
-This tutorial targets ROS 2 Humble.
-:::
-
-Install [ROS 2 Humble](https://docs.ros.org/en/humble/Installation/Ubuntu-Install-Debians.html), [Beluga](../getting-started/installation.md), and [Nav2](https://docs.nav2.org/getting_started/index.html#installation).
+### Clone `beluga-demos` repository
 
-## Create the workspace
+The [beluga-demos](https://github.com/Ekumen-OS/beluga-demos) public repository contains the `beluga_demo_nav2_integration` package, among others.
 
-Since we are using ROS 2, we need to create a workspace. Open a terminal and run the following command:
+Clone it into a workspace:
 
 ```bash
 mkdir -p ~/beluga_demo_ws/src
-```
-
-## beluga_demo_nav2 package
-
-### Clone beluga_demo
-
-The [beluga_demo](https://github.com/Ekumen-OS/beluga-demos) is a public repository where you can find the `beluga_demo_nav2` package, among others.
-
-Clone the [`Ekumen-OS/beluga-demos`](https://github.com/Ekumen-OS/beluga-demos) repository in your workspace:
-
-```bash
 cd ~/beluga_demo_ws/src
 git clone https://github.com/Ekumen-OS/beluga-demos.git
 ```
 
 For this tutorial, we will only use the `beluga_demo_nav2` package in that repository. This package contains all the necessary files to launch the simulation.
 
-### Explanation
+### Check  `beluga_demo_nav2_integration` package
 
 Since `beluga_amcl` is feature-wise compatible with `nav2_amcl` (as thoroughly discussed in the [`beluga_amcl` ROS 2 reference page](../packages/beluga_amcl/docs/ros2-reference.md)), we can just swap `beluga_amcl` for `nav2_amcl`:
```

Comment:
We can remove the "(as thoroughly discussed in the `beluga_amcl` ROS 2 reference page)" part

Reply:
I'd still want to point folks to the slight difference that exist. Ideas for other phrasings are welcome.

Reply:
That's fine by me.